root/documentation/compara_grch37.conf

```diff
@@ -0,0 +1,250 @@
+<endpoints>
+  <homology_symbol>
+    description=Retrieves homology information (orthologs) by symbol
+    endpoint=homology/symbol/:species/:symbol
+    method=GET
+    group=Comparative Genomics
+    output=json
+    output=xml
+    output=orthoxml
+    <params>
+      <species>
+        type=String
+        description=Species name/alias
+        example=__VAR(species)__
```

Comment:
These presumably require updates to ensembl-rest-deploy?

Reply:
This will need changes to rest_private which stores the respective conf files for deployment. PR for the rest_private changes will be done later.